src/__tests__/upsert.test.ts

```diff
@@ -38,6 +38,7 @@ describe('Upsert Integration Tests', () => {
 			...{
 				operation: 'UPSERT',
 				title: 'GH Action Test Identity',
+				icon: 'GH Action Test Identity',
```

Comment:
Shouldn't this match an actual icon that exists in the system? or will this not matter?

Reply:
Does not matter, port ignores icon validation